spec/rspec/matchers/built_in/output_spec.rb

```diff
@@ -194,5 +194,26 @@ def print_to_stream(msg)
         expect(output("foo").description).to eq('output "foo" to some stream')
       end
     end
+
+    RSpec.describe "can capture stdin and stderr" do
+      it "prints diff for both when both fail" do
+        expect {
+          expect { print "foo"; $stderr.print("bar") }
+            .to output(/baz/).to_stdout
+            .and output(/qux/).to_stderr
+        }
+          .to fail_including(
```

Comment:
I think this is what is breaking the built, it seems antiquated [because it is] but 1.8.7 required:
`thing.
to(`
over:
`thing
.to(`